### PR TITLE
Exploration Vendor Additions

### PR DESCRIPTION
**Pull request by MarkusLarsson421**
## About The Pull Request
Removes unused and irrelevant items in the vendor and adds items to help Explorers not die as easily. Look in Screenshots&Videos for a picture.

## Why It's Good For The Game
Helps Explorers not get fucked over by only having one method of getting to and from the station.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

image

image

These items are available in the emergency equipment storage container on the emergency capsule and on the Rad and Echo mining shuttles.

image

</details>

## Changelog
:cl:
add: Added Exploration Emergency Shuttle Capsule.
add: Added Exploration Emergency Equipment Storage.
del: Removed unused items from the Exploration Vendor.
balance: Changed the Exploration Vendor to include Lazarus Injector and Tracking Implant Kit like the Miners' Vendor.
balance: Added multiple methods to the Exploration Vendor to help Explorers not die in outer space as easily.
tweak: Changed the pod safes to be their own subcategory under `/obj/item/storage/pod/`.
balance: Tweaked the contents of the pod storage compared to the new exploration variant.
tweak: [Echo, Rad] Changed the contents of the mining shuttles' storage container.
/:cl:

### CODE REVIEW

File: code/modules/exploration_crew/exploration_vendor.dm [hunk 1]
```diff
@@ -11,26 +11,29 @@
...
-		new /datum/data/vendor_equipment("Burn Healing Kit",			/obj/item/storage/firstaid/fire,									3000),
+		new /datum/data/vendor_equipment("Emergency capsule pod",		/obj/item/survivalcapsule/shuttle/traitor,							2500),
 		new /datum/data/vendor_equipment("Advanced Healing Kit",		/obj/item/storage/firstaid/advanced,								5000),
 		new /datum/data/vendor_equipment("Explorer's Webbing",			/obj/item/storage/belt/mining,										2000),
-		new /datum/data/vendor_equipment("Breaching Charge",			/obj/item/grenade/exploration,										1000),
-		new /datum/data/vendor_equipment("Charge Detonator",			/obj/item/exploration_detonator,									10000),
-		new /datum/data/vendor_equipment("Multi-Purpose Energy Gun",	/obj/item/gun/energy/e_gun/mini/exploration,						20000),
-		new /datum/data/vendor_equipment("Expanded E. Oxygen Tank",		/obj/item/tank/internals/emergency_oxygen/engi,						1000),
-		new /datum/data/vendor_equipment("Survival Knife",				/obj/item/knife/combat/survival,							1000),
+	//Novelty items
```
Comment (Tsar-Salat) [inline code/modules/exploration_crew/exploration_vendor.dm:28]: It's preferable to use the vendor categorization feature if you are going to try to categorize the vendor by item type, rather than by comments.

See the clothesmate if you need a reference.

**Review by Tsar-Salat — DISMISSED** [review]

File: code/modules/exploration_crew/exploration_vendor.dm (hunk 1 above)
Comment (MarkusLarsson421) [reply code/modules/exploration_crew/exploration_vendor.dm:28]: The mining one doesn't use it either, and don't think there is enough items to require it.

Comment (PowerfulBacon) [reply code/modules/exploration_crew/exploration_vendor.dm:28]: Agree with Tsar-Salat, even though I don't like the formatting of the vendor categories. There are enough items to make a long ass scrollbar, and most of this stuff is useless and can be filtered out anyway.

Comment (MarkusLarsson421) [reply code/modules/exploration_crew/exploration_vendor.dm:28]: Exploration Vendors are implemented differently than the regular vendors, I won't be able to easily add categories to them.

**Comment by MarkusLarsson421** [comment]
Ready.

**Comment by MarkusLarsson421** [comment]
ready

**Comment by MarkusLarsson421** [comment]
It's been ready for three weeks now.

**Comment by EvilDragonfiend** [comment]
🔑 I don't want this to be an easy way to get a drivable shuttle.

**Comment by Rukofamicom** [comment]
Due to personal circumstances, author has requested that all PRs be treated "as-is" for the time being. 

There are no outstanding issues with this PR so it is being left open. 
If changes are going to be requested, simply close the PR instead.

**Comment by Tsar-Salat** [comment]
> I don't like this to be an easy way to get a drivable shuttle.

I agree with dragon's sentiment, but he doesn't word it correctly.

Supercruise is part of the codebase, I really don't worry about 'limiting' peoples ability to use it or shuttles being too easily obtainable. We want people to engage with it, not a mindset that it should be dripfed. 
Bacon designed it perhaps too much with the idea that people would use it more, given the research tech locked away in disks.


So the problem here is the team gets separated or left behind, and they have to get back somehow. And the proposed solution is to add disposable emergency shuttles? 

I find this oxymoronic. This *isn't* lavaland. The purpose of the deployable structures is to provide recuperation and protections against the storm. (and a money sink at later levels for miners) You don't get to pack it up, it stays there.

But for these deployable shuttles, their purpose is to bring you home, and then what? You regroup with your team, and go back to the better shuttle and the disposable one gets parked.

And if this situation happens again? These one-and-done shuttles make no sense from a design perspective.

The fulton device is geared for reusability. I dont see the benefits of these shuttles over fultons.

tl;dr We already have a tool for doing this, I don't know why we would invent a worse wheel

:key:
